CONTRIBUTING.md

```diff
@@ -69,3 +69,67 @@ going through:
 
 - Getting started: http://rogerdudler.github.io/git-guide/
 - Understanding git (important!): https://youtu.be/1ffBJ4sVUb4
+
+## Labs
+
+This section will cover any contribution guidelines specific to
+[Labs](meta/labs).
+
+### Case Studies
+
+Contribution flow for submitting [case studies](case_studies/):
+
+1. Go to [Cloud9](https://c9.io/) and open your "hackedu" workspace.
+2. If you're not already on the `master` branch, switch to it.
+
+        $ git checkout master
+
+3. Update your hackEDU fork.
+
+        $ git pull https://github.com/hackedu/hackedu.git
+
+4. Create a new branch to add the case study to. Branch names should be
```

Comment:
Explain how to name branches and that they should reflect what they are changing.